rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/api/CoordinatorApiIT.java

```diff
@@ -166,7 +166,7 @@ public void getAllLRAs() {
             Assert.assertEquals("Expected that the call succeeds, GET/200.", Status.OK.getStatusCode(), response.getStatus());
             Assert.assertEquals("Provided API header, expected that one is returned",
                     version, response.getHeaderString(LRA_API_VERSION_HEADER_NAME));
-            data = response.readEntity(new GenericType<List<LRAData>>() {});
+            data = response.readEntity(new GenericType<List<LRAData>>() { });
```

Comment:
Is there a rule that will avoid this (since {} is  normal for this kind of statement)?

Reply:
+1, I found the rule

Reply:
Nice